prov/net/src/xnet_progress.c

```diff
@@ -958,6 +1078,10 @@ int xnet_progress_wait(struct xnet_progress *progress, int timeout)
 {
 	struct ofi_epollfds_event event;
 
+	/* We cannot enter blocking if io_uring has entries
+	 * that need submission. */
+	assert(xnet_io_uring_needs_submit(&progress->tx_io_uring));
+	assert(xnet_io_uring_needs_submit(&progress->rx_io_uring));
```

Comment:
The assert and comment look like opposites

Reply:
aargh, the assert is wrong. Thanks for catching that.